Podcast/UserSubscriptionsEndpointRequests.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  UserSubscriptionsEndpointRequest.swift
+//  Podcast
+//
+//  Created by Jack Thompson on 8/26/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class FetchUserSubscriptionsEndpointRequest: EndpointRequest {
+    
+    var userID: String
+    
+    init(userID: String) {
+        
+        self.userID = userID
+        
+        super.init()
+        
+        path = "/subscriptions/users/\(userID)/"
+        
+        httpMethod = .get
+    }
+    
+    override func processResponseJSON(_ json: JSON) {
+        let series = json["data"]["subscriptions"].map{ jsons -> Series in
+            let s = Cache.sharedInstance.update(seriesJson: jsons.1["series"])
```

Comment:
more descriptive variable name than `s`

Reply:
can't you just do ` return Cache.sharedInstance.update...`

Reply:
Yeah I don't think you even return. This code used to do some weird casting stuff but now it's unnecessary